### DISCUSSION TITLE
Recommendations for working with R tutorials in Positron?

### DISCUSSION THREAD

**Discussion by davidkane9** [posit-dev/positron#5635] [Q&A] [answered]
I make extensive use of R tutorials when teaching introductory statistics. The Tutorial Pane in RStudio works well. Is there an equivalent in Positron? If not (and I certainly can't find one), what approach do people recommend? Again, I am teaching beginner students.

**Comment by juliasilge** [comment, marked as answer]
We don't currently have a specific tutorials pane, but you can run any learnr tutorial from the R console and interact with it then in our Viewer pane. For example, with:

```r
learnr::run_tutorial("hello", package = "learnr")
```

You will get:

![learnr-tutorial](https://github.com/user-attachments/assets/4c793c6a-d24c-4ebf-adc5-089ae086158c)

**Comment by davidkane9** [comment]
Thanks for the response. Commands like that are tricky for beginner students, many of whom have never typed anything in a Console.

**Reply by juliasilge** [reply]
Makes sense! As of today, I would say that RStudio is a better fit for students who aren't comfortable typing in the console but certainly in the longer term we are interested in how Positron can best be used for teaching.
